purchase_lead_hour/models/product_supplierinfo.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ProductSupplierinfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    delay = fields.Float()
+
+    def get_next_availability_date(self):
```

Comment:
The function name doesn't really match with its content :confused: 
Suggestion: `get_date_to_compute_delay` ?